api/queries_pr.go

```diff
@@ -123,6 +101,14 @@ type ReviewRequests struct {
 	TotalCount int
 }
 
+func (r ReviewRequests) Logins() []string {
+	logins := make([]string, 0, len(r.Nodes))
+	for _, a := range r.Nodes {
+		logins = append(logins, a.RequestedReviewer.Login)
```

Comment:
Nit: if you are copying items from a struct into another struct of the same size, you can immediately allocate the new struct at its full length and avoid using `append`:
```suggestion
	logins := make([]string, len(r.Nodes))
	for i, a := range r.Nodes {
		logins[i] = a.RequestedReviewer.Login
```
This is slightly more efficient in general.